docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -1,135 +1,135 @@
 ---
-title: How to Open a Pull Request
+title: Como Abrir uma Pull Request
 ---
 
-A big part of contributing to open source is submitting changes to a project: improvements to source code or tests, updates to docs content, even typos or broken links. This doc will cover what you need to know to **open a pull request** in Gatsby.
+Uma boa parte de contribuição para open source consiste em enviar alterações para um projeto: melhorias no código-fonte ou nos testes, atualizações no conteúdo da documentação, ou até mesmo erros de digitação ou links quebrados. Este documento abordará o que você precisa saber para **abrir uma pull request** no Gatsby.
 
-## What is a Pull Request (PR)?
+## O que é uma Pull Request (PR)?
 
-In case you aren't familiar, here's how the folks at GitHub [define a pull request](https://help.github.com/en/articles/about-pull-requests):
+Caso você não esteja familiarizado, veja como o pessoal do GitHub [define uma pull request](https://help.github.com/pt/articles/about-pull-requests):
 
-> Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
+> As pull requests permitem que você informe outras pessoas sobre as alterações das quais você fez push para um branch em um repositório no GitHub. Depois que uma pull request é aberta, você pode discutir e revisar as possíveis alterações com colaboradores e adicionar commits de acompanhamento antes que as alterações sofram merge no branch base.
```

Comment:
Alterar artigo "As" => "Os"
Alterar artigo "uma" => "um"
Alterar artigo "um branch" => "uma branch"

```suggestion
> Os pull requests permitem que você informe outras pessoas sobre as alterações das quais você fez push para uma branch em um repositório no GitHub. Depois que um pull request é aberto, você pode discutir e revisar as possíveis alterações com colaboradores e adicionar commits de acompanhamento antes que as alterações sofram merge na branch base.
```

Reply:
Essa é uma citação, eu trouxe o texto na integra:
![image](https://user-images.githubusercontent.com/5200384/68024174-1768b280-fc88-11e9-8b7f-d4e544d33e19.png)
https://help.github.com/pt/github/collaborating-with-issues-and-pull-requests/about-pull-requests

Devo alterá-lo mesmo assim?

Reply:
https://github.com/gatsbyjs/gatsby-pt-BR/pull/29/commits/3b3c21095fb45f6be256aa235cc1b8677bc185fa
https://github.com/gatsbyjs/gatsby-pt-BR/pull/29/commits/b497fb691d6b7d2acfce84e6cf0851f882f05cf3